content-src/components/Search/Search.jsx

```diff
@@ -154,7 +154,7 @@ export class _Search extends React.PureComponent {
               data-l10n-id="newtab-search-box-search-the-web-input"
               ref={this.onSearchHandoffButtonMount}
               onClick={this.onSearchHandoffClick}
-              tabIndex="-1"
+              tabIndex="0"
```

Comment:
Was this set to -1 to begin with to avoid some problems of it getting focus?

Reply:
The focus was turned off for an experiment, I think.

Reply:
If it were turned off for an experiment, then why? If it's because there are bugs, then we should fix them before changing it from -1. Does this even work correctly with tabindex=0?

Reply:
It was turned off while the search handoff to the awesomebar was an experiment, I believe - I got the go-ahead from the folks that worked on it that it should be ready to turn back on. It just puts it back into the regular tab order for the page :)

https://bugzilla.mozilla.org/show_bug.cgi?id=1544527

Reply:
Where in the bug does it say tabindex=-1 was for the experiment and it's good to set to tabindex=0?